vprdb/io/reader.py

```diff
@@ -0,0 +1,84 @@
+import mrob
```

Comment:
It is more common to have `read_utils` if file contains not class but only functions, when I see `reader` I expect some interface usually.

Reply:
The same about exporter